groups/api/http/decode.go

```diff
@@ -68,6 +68,21 @@ func DecodeGroupUpdate(_ context.Context, r *http.Request) (interface{}, error)
 	return req, nil
 }
 
+func decodeUpdateGroupTags(_ context.Context, r *http.Request) (interface{}, error) {
```

Comment:
I noticed the other functions in this file are exported. You can check if that is required